app/src/main/res/values/styles.xml

```diff
@@ -15,12 +15,44 @@
         <item name="android:background">?selectableItemBackground</item>
         <item name="android:drawableStart">@drawable/ic_info</item>
         <item name="android:drawableEnd">@drawable/ic_keyboard_arrow_right</item>
-        <item name="android:drawablePadding">16dp</item>
+        <item name="android:drawablePadding">@dimen/alert_banner_drawable_padding</item>
         <item name="android:gravity">center_vertical</item>
-        <item name="android:textSize">18sp</item>
-        <item name="android:padding">32dp</item>
+        <item name="android:textSize">@dimen/alert_banner_text_size</item>
+        <item name="android:padding">@dimen/alert_banner_padding</item>
         <item name="android:textStyle">bold</item>
         <item name="android:textColor">@color/white</item>
         <item name="fontFamily">@font/montserrat</item>
     </style>
+    
+    <style name="BigButton" parent="Widget.MaterialComponents.Button.OutlinedButton">
+        <item name="android:layout_width">match_parent</item>
+        <item name="android:layout_height">@dimen/big_button_height</item>
+    </style>
+
+    <style name="BigButton.Blue" parent="BigButton">
+        <item name="backgroundTint">@color/bluejay</item>
+        <item name="android:textColor">@color/white</item>
+    </style>
```

Comment:
what if instead of defining these two styles, we define `button_height=56` in dimens.xml, change `colorPrimary=@color/bluejay`? In the end we will have this:

```xml
<!-- Outlined button -->
<Button
  style="?attr/materialButtonOutlinedStyle"
  android:layout_width="match_parent"
  android:layout_height="@dimen/button_height" />

<!-- Blue button -->
<Button
  android:layout_width="match_parent"
  android:layout_height="@dimen/button_height" />
```

Reply:
Since the button height is a standardized part of all the button elements in the app, it should probably be in the style. You can always override the height if you need a button to be a non-standard height somewhere. I'm trying to get the button elements as close to a custom view as possible without defining a view. Ideally, on any button, the only parameters added should be `text` and `marginTop` (at least while we are using LinearLayouts everywhere).

What value is there in manually declaring the same width and height all over the layout files?

Reply:
Okay. But let's rename them to `Button`, and `OutlinedButton`?
```xml
<style name="ColoredButton">
  <item name="android:layout_width">match_parent</item>
  <item name="android:layout_height">@dimen/big_button_height</item>
</style>

<style name="OutlinedButton" parent="Widget.MaterialComponents.Button.OutlinedButton">
  <item name="android:layout_width">match_parent</item>
  <item name="android:layout_height">@dimen/big_button_height</item>
</style>
```

Reply:
Sounds good